### PR TITLE
Adds Long Blazers

### PR DESCRIPTION
**Pull request by WickedCybs**
I've tweaked the existing blazer sprite, lengthening it for those who want a longer coat in the same style. This puts the blazers in its own selection as well, so unfortunately anyone currently using the blazers in the loadout will have to reselect it.

A comparison of the lengthened variant on the left with the usual variant on the right while open.
image

### CODE REVIEW

**Comment by WickedCybs** [comment]
!review

**Review by Casper3667 — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 1]
```diff
@@ -161,10 +161,17 @@ datum/gear/suit/colorvest
...
 
 /datum/gear/suit/blazer
-	display_name = "blazer"
+	display_name = "blazer selection"
 	path = /obj/item/clothing/suit/storage/toggle/suitjacket/blazer
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
+/datum/gear/suit/blazer/New()
+	..()
+	var/blazers = list()
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:170]:
```suggestion
	var/list/blazers = list()
```

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 2]
```diff
@@ -161,10 +161,17 @@ datum/gear/suit/colorvest
...
+	blazers["blazer"] = /obj/item/clothing/suit/storage/toggle/suitjacket/blazer
+	blazers["long blazer"] = /obj/item/clothing/suit/storage/toggle/suitjacket/blazer/long
+	gear_tweaks += new/datum/gear_tweak/path(blazers)
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:173]:
```suggestion
	gear_tweaks += new /datum/gear_tweak/path(blazers)
```

File: code/modules/clothing/suits/jobs.dm [hunk 3]
```diff
@@ -279,6 +279,14 @@ obj/item/clothing/suit/apron/overalls/blue
 	item_state = "blazer"
 	opened = TRUE
 
+/obj/item/clothing/suit/storage/toggle/suitjacket/blazer/long
+	name = "long blazer"
+	desc = "A charming long jacket."
+	desc_fluff = "for when you want to play ball sports like an aristocrat."
```
Comment (Geevies) [inline code/modules/clothing/suits/jobs.dm:285]:
```suggestion
	desc_fluff = "For when you want to play ball sports like an aristocrat."
```

**Review by Geevies — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 2 above)
Comment (WickedCybs) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm:173]: Is the single space here intentional?

**Pull request merged**